### PR TITLE
Set up linter commands, with instructions

### PR DESCRIPTION
**Pull request by melanierichards**
## What this PR does

Adds CSS linter and documents the use of said linters

### CODE REVIEW

**Comment by melanierichards** [comment]
FYI @molant I fixed my merge conflicts

**Comment by molant** [comment]
@melanierichards there are issues with JS now that we've updated to the latest eslint. Might be as easy as running --fix. I'll take a look on Monday.

File: cdp.pid [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (alrra) [inline cdp.pid:1]: This file should be removed, and  `*.pid` should be added to the `.gitignore` file.

File: .sonarrc [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (alrra) [inline .sonarrc:1]: This file should be renamed to `.sonarwhalrc`.

Possible TODO: Integrate `sonarwhal`.

**Comment by melanierichards** [comment]
We made a new PR for this, closing this old one

**Pull request closed without merging**